### PR TITLE
SRVKS-711: Add note about routes

### PR DESCRIPTION
**Pull request by abrennan89**
Jira: https://issues.redhat.com/browse/SRVKS-711
Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1941877

Applies for OCP 4.6+

Direct preview link: https://deploy-preview-34532--osdocs.netlify.app/openshift-enterprise/latest/serverless/knative_serving/serverless-configuring-routes?utm_source=github&utm_campaign=bot_dp

### CODE REVIEW

**Review by cardil — APPROVED** [review]
/lgtm

from QE

File: modules/serverless-openshift-routes.adoc [hunk 1]
```diff
@@ -1,7 +1,12 @@
 [id="serverless-openshift-routes_{context}"]
 = Configuring {product-title} routes for Knative services
 
-If you want to configure a Knative service to use your TLS certificate on {product-title}, you must disable the automatic creation of a route for the service by the {ServerlessOperatorName}, and instead manually create a `Route` resource for the service.
+If you want to configure a Knative service to use your TLS certificate on {product-title}, you must disable the automatic creation of a route for the service by the {ServerlessOperatorName}, and instead manually create a route for the service by using the following procedure.
```
Comment (kalexand-rh) [inline modules/serverless-openshift-routes.adoc:4]:
```suggestion
If you want to configure a Knative service to use your TLS certificate on {product-title}, you must disable the automatic creation of a route for the service by the {ServerlessOperatorName} and instead manually create a route for the service.
```

> Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes". Instead, focus on the purpose or value of the content, and include words that users are likely to search for. 

I think that if you're going to explicitly refer to the steps that the reference in the note makes more sense.

**Review by kalexand-rh — COMMENTED** [review]
This is looking good! Please remember to squash your commits. :)

**Review by kalexand-rh — COMMENTED** [review]
LGTM!

**Pull request merged**